lib/domain.js

```diff
@@ -416,9 +446,19 @@ EventEmitter.usingDomains = true;
 
 const eventInit = EventEmitter.init;
 EventEmitter.init = function() {
-  this.domain = null;
+  Object.defineProperty(this, 'domain', {
+    configurable: true,
+    enumerable: false,
+    value: null,
+    writable: true
+  });
   if (exports.active && !(this instanceof exports.Domain)) {
-    this.domain = exports.active;
+    Object.defineProperty(this, 'domain', {
+      configurable: true,
+      enumerable: false,
+      value: exports.active,
+      writable: true
+    });
```

Comment:
This reassigns the value. Therefore it should be sufficient to revert to the code as it was before (`this.domain = exports.active;`).